### PR TITLE
made relevant changes to home page

### PR DESCRIPTION
**Pull request by blbudima**
very cool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # build-a-website
 
-This repository holds the syllabus and lesson plans for Mar Vista HS "build a website" course.
-This is still a work in progress.
+This repository holds the syllabus and lesson plans for Mar Vista High SChool "Build a Website" program.
```
Comment (wes-chen) [inline README.md]: capitalization in School

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 ## Syllabus
 
-This is a course that will take place over 12 weeks.
+This is a program that will teach you how to create your own website.
 
-The first two weeks will deal with making your own Github account, with Github Desktop, and Github Pages.
+The first two lessons will cover the essentials with Git, Github, Github Desktop, and Github Pages.
 
-The next ten weeks will cover basic fundamentals of HTML, CSS, and a little Javascript.
+The next ten lessons will cover basic fundamentals of HTML, CSS, and a little bit of Javascript. That is all you will need to creat your website.
```
Comment (wes-chen) [inline README.md]: spelling create xd

**Comment by blbudima** [comment]
changes made
do i have to do another pull request or smth or do u just approve it in this pr

**Comment by wes-chen** [comment]
as long as you committed this pr is still good

**Pull request merged**